sched/scheduler/cluster_state.go

```diff
@@ -84,6 +84,7 @@ func (ns *nodeState) ready() bool {
 
 // Stars a goroutine loop checking node readiness, waiting 'backoff' time between checks, and exiting if node is fully removed.
 func (ns *nodeState) startReadyLoop(rfn ReadyFn) {
+	ns.readyCh = make(chan interface{})
```

Comment:
the channel is created when ns is created in newNodeState() - remove from here or from newNodeState()?